src/layout/right-pane/RightPane.module.css

```diff
@@ -0,0 +1,10 @@
+.rightpane {
+  display:flex;
+  justify-content: center;
+  align-items: start;
+  height:100%;
+  width:100%;
+  padding-top: 9px;
+  background-color: lightgray;
+  overflow-y: scroll;
```

Comment:
If you use `scroll` even page doesn't need any scrolling, it will show empty scrollbars. But if you use `auto` it won't show scroll bars if they not needed.